.github/workflows/main.yml

```diff
@@ -0,0 +1,26 @@
+name: AI Code Reviewer
+
+on: [pull_request]
+
+    # types:
+    #   - opened
+    #   - synchronize
+# permissions: write-all
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      pull-requests: write
+    steps:
+      - name: Checkout Repo
+        uses: actions/checkout@v3
+
+      - name: AI Code Review Action
+        uses: freeedcom/ai-codereviewer@v2.6.0
+        with:
+          GITHUB_TOKEN: ${{ secrets.GH_TOKEN }}
+          OPENAI_API_KEY: ${{ secrets.OPENAI_API_KEY }}
+          OPENAI_API_MODEL: "gpt-4" # Optional: defaults to "gpt-4"
+          exclude: "**/*.json, **/*.md" # Optional: exclude patterns separated by commas
+          post-if-error: 'true'
```

Comment:
The `post-if-error` field has been added. Make sure this is a valid input for the `ai-codereviewer` action, as it's not mentioned in the pull request description.